JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -211,9 +211,20 @@ public function denormalize($data, $class, $format = null, array $context = [])
             $overrideClass = false;
         }
 
+        $instanceClass = $overrideClass ? get_class($context['object_to_populate']) : $class;
+        $reflectionClass = new \ReflectionClass($instanceClass);
```

Comment:
Using reflection at runtime looks like a bad idea. It will considerably decrease performances.
We should store if the class is abstract or not in the `ClassMetadata` so it will be handled by the cache warmer and it will not have a negative impact at runtime.